src/settings/CountrySearch.ts

```diff
@@ -0,0 +1,16 @@
+import { makeDebounce } from "../shared/utils";
+import "./CountrySearch.css";
+
+const debounce = makeDebounce();
+
+export class CountrySearch {
```

Comment:
This class does not actually perform the search of the countries, like its name said. I think we can solve this in one of two ways:

1. Abstract the class of the countries, by just renaming it to something like `DebouncedTextInput`, and keep the class the same.
2. Add search logic here, so the instance will call the `onChange` callback with the codes of filtered countries.